test-e2e/utils/payments.ts

```diff
@@ -63,18 +63,30 @@ export async function finishAndCheckSubscription(I: CodeceptJS.I, billingDate: D
   I.see(yearlyPrice);
   I.see('/year');
   I.see('Next billing date is on ' + formatDate(billingDate));
-  I.see('Cancel subscription');
 
-  I.waitForElement('[class*="transactionItem"]');
+  if (providerName?.includes('JW')) {
```

Comment:
what do you think of including additional param `canChangeSubscription` or something, instead of using provider name?